src/PowerPlaywright.Framework/Controls/Pcf/IEmailControl.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PowerPlaywright.Framework.Controls.Pcf
+{
+    using PowerPlaywright.Framework.Controls.Pcf.Attributes;
+    using PowerPlaywright.Framework.Controls.Pcf.Classes;
+
+    /// <summary>
+    /// An interface for the MscrmControls.FieldControls.Email control.
+    /// </summary>
+    [PcfControl("MscrmControls.FieldControls.EmailAddressControl")]
+    public interface IEmailControl : ISingleLineEmail
```

Comment:
For consistency, lets keep the interface name the same as the control name:

```suggestion
    public interface IEmailAddressControl : ISingleLineEmail
```

Reply:
Agreed